src/renderers/shared/fiber/__tests__/ReactIncrementalScheduling-test.js

```diff
@@ -100,7 +567,7 @@ describe('ReactIncrementalScheduling', () => {
     expect(ReactNoop.getChildren('c')).toEqual([span('c:2')]);
   });
 
-  it('works on roots with animation priority before deferred work', () => {
+  it('handles interleaved lo-pri and hi-pri work', () => {
```

Comment:
>https://github.com/facebook/react/blob/master/src/renderers/shared/fiber/ReactPriorityLevel.js#L21

Sorry, I meant that it is not used anywhere in the code or tests. It wasn't clear to me that it works, I thought it's just something added for the future when priority stuff is more fleshed out.

OK, I'll change it back to animation. But we'll need to rename all of them in the future anyway.